src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -2876,7 +2876,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>Members of readonly field '{0}' cannot be used as a ref or out value (except in a constructor)</value>
   </data>
   <data name="ERR_AssgReadonly" xml:space="preserve">
-    <value>A readonly field cannot be assigned to (except in a constructor or a variable initializer)</value>
+    <value>A readonly field cannot be assigned to (except in the constructor of the class in which the field is defined or a variable initializer))</value>
```

Comment:
There may be more than one constructor so this should probably remain as "except in a constructor …"

Reply:
Done.